robotouille/robotouille_simulator.py

```diff
@@ -1,33 +1,52 @@
 import pygame
+import time
 from utils.robotouille_input import create_action_from_control
 from robotouille.robotouille_env import create_robotouille_env
+from backend.movement.player import Player
+from backend.movement.movement import Movement, Mode
 
 
 def simulator(environment_name: str, seed: int=42, noisy_randomization: bool=False):
     # Your code for robotouille goes here
-    env, json, renderer = create_robotouille_env(environment_name, seed, noisy_randomization)
+    movement_mode = Mode.TRAVERSE
```

Comment:
make this an argument passed in my command line and make the default mode `IMMEDIATE` so that we can tune the movement later to the cooking time when the game is in ready state.

Reply:
Edited accordingly